src/pointer-analysis/value_set.h

```diff
@@ -82,7 +82,12 @@ class value_sett
     size_t size() const { return data.size(); }
     bool empty() const { return data.empty(); }
 
+    void erase(unsigned i) { data.erase(i); }
+    void erase(iterator it) { data.erase(it); }
```

Comment:
More important: use `const_iterator` (`iterator` is convertible to `const_iterator`, but the reverse is not true)